frontend/src/app/AppRouter.tsx

```diff
@@ -51,11 +51,11 @@ export const AppRouter = (): JSX.Element => {
         />
         <Route
           path={PRIVACY_POLICY_ROUTE}
-          element={<PublicElement strict element={<PrivacyPolicyPage />} />}
```

Comment:
Just for my reference, what does `strict` controls? Is it [this](https://reactjs.org/docs/strict-mode.html)?

Reply:
No sorry, it's this: https://github.com/opengovsg/FormSG/blob/form-v2/develop/frontend/src/app/PublicElement.tsx#L15

on hindsight `strict` is a bad name for that prop haha